OSD-15452 : Updating Readme (#42)
openshift/backplane-cli: README.md

[![codecov](https://codecov.io/gh/openshift/backplane-cli/branch/main/graph/badge.svg)](https://codecov.io/gh/openshift/backplane-cli)
[![License](https://img.shields.io/:license-apache-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0.html)

backplane-cli is a CLI tool to interact with [backplane api](https://github.com/openshift/backplane-api).

The onboarding documentation can be found on [The Source Wiki](https://source.redhat.com/groups/public/openshiftplatformsre/wiki/backplane_user_documentation).

## Install

### Option 1: Use `go install`

For using this method, you need to have `go` installed.

The minimal version required is `go 1.18`. You can check this with `go version`.

Then run `go install github.com/openshift/backplane-cli/cmd/ocm-backplane@{TAG_VERSION}`
where `TAG_VERSION` is the version you wish to install.

Example: `go install github.com/openshift/backplane-cli@0.0.8`

> `go install` will fetch, build binary and install that to your $GOBIN (if set) or $GOPATH/bin,

Reviewer: ```suggestion
`go install` will fetch, build the binary and install it to your $GOBIN (if set) or $GOPATH/bin,
```

> you should move this binary onto your $PATH if desired.

Reviewer: This won't be necessary, usually, if go environment structure is configured correctly, `GOBIN` will add the binaries to `PATH`
```suggestion
```

```
$ ocm backplane version

> 0.0.8

Reviewer: ```suggestion
0.0.35
```

  > Note: Load the console plugin from backplane-cli is not sufficient to access the console plugin,
  backplane-api to expose the console plugin service explicitly is needed.

## Cloud Console

- Login to the target cluster via backplane as the above.
- Run the below command and it will launch the cloud console of the current logged in cluster.
  ```
  $ ocm backplane cloud console
  Console Link:
  Link: https://xxxxx
  ```
- Follow the above link to access the console.

  #### Open in browser

>   You can directly open the console in browser by adding flag -b or setting environment variable `BACKPLANE_DEFAULT_OPEN_BROWSER=true`. Example,

Reviewer: ```suggestion
 You can directly open the console in the browser by adding flag `-b` or setting the environment variable `BACKPLANE_DEFAULT_OPEN_BROWSER=true`.
```
